models/engine/db_storage.py

```diff
@@ -74,3 +74,12 @@ def reload(self):
     def close(self):
         """call remove() method on the private session attribute"""
         self.__session.remove()
+
+    def get(self, cls, id):
+        """retrieve the object identified with class.id """
+        k = "{}.{}".format(cls.__name__, id)
```

Comment:
Please  specify what do you mean by naming that variable "k". Is it key?

Reply:
sure, let me change it